chewie/radius_datatypes.py

```diff
@@ -189,22 +189,22 @@ def parse(cls, packed_value):
         return cls(struct.unpack("!%ds" % len(packed_value), packed_value)[0])
 
     def pack(self, attribute_type):
+
+        def chunks(l):
```

Comment:
let's use better variable names here - even the word "data" is more meaningful than the letter L